storage/leveldb/leveldb.go

```diff
@@ -30,15 +30,16 @@ type DB struct {
 
 // NewDB is a constructor for the leveldb persister
 // It creates the files in the location given as parameter
-func NewDB(path string, batchDelaySeconds int, maxBatchSize int) (s *DB, err error) {
+func NewDB(path string, batchDelaySeconds int, maxBatchSize int, maxOpenFiles int) (s *DB, err error) {
 	err = os.MkdirAll(path, rwxOwner)
 	if err != nil {
 		return nil, err
 	}
```

Comment:
Should there be a verification against a minimum possible number for `maxOpenFiles`? Would `leveldb` work well when `OpenFilesCacheCapacity = 0`?

Reply:
Will test to see the appropriate boundaries.

Reply:
Done.